gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageOptions.java

```diff
@@ -88,6 +88,12 @@
   /** Default setting for GCS HTTP request headers. */
   public static final ImmutableMap<String, String> HTTP_REQUEST_HEADERS_DEFAULT = ImmutableMap.of();
 
+  /** Default setting for authorization provider. */
```

Comment:
Use the AuthorizationHandler class itself not string, as the type of this field here directly